datarepo-clienttests/src/main/java/utils/DataRepoUtils.java

```diff
@@ -34,7 +34,7 @@ public static JobModel waitForJobToFinish(RepositoryApi repositoryApi, JobModel
     job = repositoryApi.retrieveJob(job.getId());
 
     while (job.getJobStatus().equals(JobModel.JobStatusEnum.RUNNING) && pollCtr >= 0) {
-      Thread.sleep(secondsIntervalToPollJob);
+      Thread.sleep(secondsIntervalToPollJob * 1000);
```

Comment:
I prefer this way to sleep: `TimeUnit.SECONDS.sleep`

Reply:
That is much better, thank you. Changed.